filters/auth/webhook.go

```diff
@@ -0,0 +1,70 @@
+package auth
```

Comment:
i think the hook should be a completely independent filter.

Reply:
I disagree, because it only makes sense for auth. As we discussed in the chat you also had no use case for non IAM webhooks.

Reply:
+1 we agreed not to make it generic